liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java

```diff
@@ -40,6 +40,12 @@ public static void cleanUpAfterClass() throws Exception {
       BaseDevTest.cleanUpAfterClass();
    }
 
+   @Test
+   /* simple double check. if failure, check parse in ci.common */
+   public void verifyJsonHost() throws Exception {
+      checkLogMessage(2000, "CWWKT0016I"); 
```

Comment:
So what does this do exactly? I thought you were going to verify the format was json.

Reply:
Maybe you should pass in the escaped String for the protocol instead. It would only find it if the log file was in json format.

Reply:
@cherylking 
I was going to do additional verification on the message format but realized that's an OpenLiberty feature.
I originally kept the whole line for verification and now verify for only the code and the hostname with the escape sequence after.